.github/actions/spelling/advice.md

```diff
@@ -7,18 +7,18 @@ If items relate to a ...
   Please add a file path to the `excludes.txt` file matching the containing file.
 
   File paths are Perl 5 Regular Expressions - you can [test](
-https://www.regexplanet.com/advanced/perl/) yours before committing to verify it will match your files.
+<https://www.regexplanet.com/advanced/perl/>) yours before committing to verify it will match your files.
 
   `^` refers to the file's path from the root of the repository, so `^README\.md$` would exclude [README.md](
 ../tree/HEAD/README.md) (on whichever branch you're using).
 
 * well-formed pattern.
 
-  If you can write a [pattern](https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-patterns) that would match it,
+  If you can write a [pattern](<https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-patterns>) that would match it,
```

Comment:
Do not include the `<>` inside the link.

Reply:
The markdown linter has an issue with "bare_urls" and recommended adding angle brackets. I will change it back but the linter may not pass. View this site for more info: https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md